pkg/tracking/store/sql/runs_internal_test.go

```diff
@@ -1,4 +1,4 @@
-//nolint:ireturn
+// nolint
```

Comment:
why does this need to change in this PR?

Reply:
I've seen this before, we should update Go and/or the linter. I believe this was fixed somewhere.